modules/http/src/main/java/org/mule/extension/http/internal/request/HttpRequestOperations.java

```diff
@@ -87,19 +85,19 @@ public void request(@Optional(defaultValue = "/") String path,
       HttpRequesterRequestBuilder resolvedBuilder = requestBuilder != null ? requestBuilder : new HttpRequesterRequestBuilder();
       UriParameters uriParameters = client.getDefaultUriParameters();
 
-      String resolvedHost = resolveIfNecessary(overrides.getHost(), uriParameters.getHost(), muleEvent);
-      Integer resolvedPort = resolveIfNecessary(overrides.getPort(), uriParameters.getPort(), muleEvent);
-      String resolvedBasePath = config.getBasePath().apply(muleEvent);
+      String resolvedHost = resolveIfNecessary(overrides.getHost(), uriParameters.getHost());
```

Comment:
you're removing the usages of event but still receiving the even in the operation